### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by marians**
Public repo for a public dockerhub-hosted Docker image for Hugo

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+**We welcome your contributions!**
+
+The best way to contribute changes are Pull Requests. Please note the following before sending PRs:
+
+* Please make one Pull Request per topic/issue. Please avoid addressing several non-related issues in a single pull request.
+
+* By sending a Pull Request, you grant us (Giant Swarm) an irrevocable and unlimited license to use the content/code you contribute without any restriction. You accept that your contribution might be published under the license of this repository. Giant Swarm is not obliged to list you as a contributor.
+
+Questions, comments and requests for additions/improvements can be addressed by [adding an issue](https://github.com/giantswarm/cli/issues/new) to this repository.
```
Comment (ZeissS) [inline CONTRIBUTING.md]: wrong url

File: README.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
-# hugo-docker
-Dockerized version of the HUGO (gohugo.io) executable
+# HUGO Website Engine in Docker
+
+**Dockerized version of the HUGO (gohugo.io) executable. Provides a quick way to run the HUGO 0.14 linux executable from a docker image.**
+
+More about Hugo: http://www.gohugo.io/
+
+Giant Swarm uses Hugo for it's [documentation](https://docs.giantswarm.io/). We provide this image basically to fulfill our own needs, but wlecome contributions and suggestions via [github.com/giantswarm/hugo-docker](https://github.com/giantswarm/hugo-docker).
```
Comment (ZeissS) [inline README.md]: wlecome

**Comment by ZeissS** [comment]
Otherwise LGTM

**Comment by marians** [comment]
Thanks @ZeissS! Fixed the things you mentioned.

**Pull request merged**